codebases/open-source-project-template.html

```diff
@@ -0,0 +1,117 @@
+---
+# SPDX-License-Identifier: CC0-1.0
+# SPDX-FileCopyrightText: 2021-2023 The Foundation for Public Code <info@publiccode.net>
+layout: codebase
+title: Codebases we work with - Open Source Project Template
+---
+
+<div class="cb-section">
+  <div>
+    <h1 class="title">Codebases we work with</h1>
+  </div>
+</div>
+
+<hr>
+
+<div class="cb-section">
+  <div class="cb-maturity-detail-header">
+    <div class="cb-maturity-detail-title cb-maturity-s2">
+      <!-- div class="cb-logo"><img src="/photos/codebases/open-source-project-template/open-source-project-template.png" alt="Open Source Project Template logo"></div -->
+      <h1>Open Source Project Template</h1>
+    </div>
+    <div class="cb-maturity-stage cb-maturity-s2">Stage 2: <strong>Incubation</strong></div>
+  </div>
+
+  <div class="cb-detail-intro">
+    <div class="cb-detail-info">
+      <h3>What it is</h3>
+      <p>A template project containing 'Community Health Files' and relevant checklists.</p>
+      <p>It's intention is to be a practical starter when releasing a project as Open Source.</p>
+      <p>It was created in a general way to be usable for anyone, and proposes well known conventions and de-facto standards when possible.</p>
+    </div>
+
+    <div class="cb-detail-gallery">
+      <a href="#img1">
+          <img class="lb-thumb" src="../photos/codebases/open-source-project-template/open-source-project-template-templates-screenshot.png" alt="Screenshot of templates contained in the open-source-project-template repository">
+      </a>
+      <!-- lightbox container hidden with CSS -->
+      <div class="lightbox" id="img1">
+        <!--<a href="#img3" class="lb-btn lb-btn-prev"></a>-->
+        <a href="#_" class="lb-btn lb-btn-close"><span class="lb-btn-text">close</span></a>
+        <a href="#img1">
+          <img src="../photos/codebases/open-source-project-template/open-source-project-template-templates-screenshot.png" alt="Screenshot of templates contained in the open-source-project-template repository">
+        </a>
+        <!--<a href="#img2" class="lb-btn lb-btn-next"></a>-->
+      </div>
+    </div>
+  </div>
+</div>
+
+<hr>
+
+<div class="cb-section">
+  <div class="cb-detail-info">
+    <h3>How we're helping</h3>
+    <p>We're helping launch the Open Source Project Template development by:</p>
+    <ul>
+      <li>Providing feedback on the contents</li>
```

Comment:
```suggestion
      <li>Providing feedback on the content</li>
```

Reply:
To me, "contents" seems correct, as the templates and other files in the repository are specific and can be listed with the `ls` command, like a table of contents ... we're not talking about something abstract or amorphous like "web content".